stories/sign-up.stories.jsx

```diff
@@ -0,0 +1,40 @@
+// https://github.com/EnCiv/civil-pursuit/issues/150
+
+import React from 'react'
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+import SignUp from '../app/components/sign-up.jsx'
+
+export default {
+  component: SignUp,
+  args: {},
+  decorators: [
+    Story => (
```

Comment:
The stories weren't working, and I was getting an error in the browser about logger not being defined.   logger is something we use instead of console so that the messages get stored in a log.  

I know it was working before but don't see why it's  changed.  But if you change the decorator to this it will work:
```
    Story => {
      if (!window.logger) window.logger = console
      return (
        <div>
          success@email.com / 'password' will succeed
          <br />
          all other combinations will fail
          <Story />
        </div>
      )
    },
```